sdk-core/src/main/java/co/reachfive/identity/sdk/core/Provider.kt

```diff
@@ -14,7 +15,8 @@ interface ProviderCreator {
     fun create(
         providerConfig: ProviderConfig,
         sessionUtils: SessionUtilsClient,
-        context: Context
+        context: Context,
+        sdkConfig: SdkConfig,
```

Comment:
Introduced `sdkConfig` there so that `scheme` can be put in the intent (used for validating the redirection).